### PR TITLE
 Added Windows installation instructions

### PR DESCRIPTION
**Pull request by pieterhelsen**
Added detailed installation instructions for Windows.
Also updated the default README.md to cross-reference to the new Windows installation instructions.

### CODE REVIEW

File: WINDOWS.md [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+```
+
+3. Create a virtual environment for `chiadog` to run in
+
+```
+# Create a new virtual environment in this folder called 'venv'
+python.exe -m venv venv
+
+# Activate the newly created virtual environment
+. ./venv/Scripts/activate
```
Comment (martomi) [inline WINDOWS.md:48]: Should this not have the opposite slashes / \ ? (I don't know, asking 😄 )
Same about the other occurrences of that.

**Review by martomi — COMMENTED** [review]
Great work, looks good!

File: WINDOWS.md (hunk 1 above)
Comment (pieterhelsen) [reply WINDOWS.md:48]: Good catch! That's what I get for typing from memory instead of copy/pasting.

**Comment by martomi** [comment]
I think I can't exclude this from the PR myself. @pieterhelsen I think you did change permissions of install.sh and start.sh scripts by mistake? Can you drop these two permission changes from the PR?

**Comment by pieterhelsen** [comment]
Looks like PyCharm on Windows automatically changes file permissions; updated.

**Review by martomi — APPROVED** [review]

**Pull request merged**